src/UICrafter.Client/Components/AppViewDesigner.razor

```diff
@@ -1,6 +1,6 @@
 @using UICrafter.Client.Components.DnDComponents
 
-<MudContainer MaxWidth="MaxWidth.Small">
+<MudContainer MaxWidth="MaxWidth.Small" @onclick:stopPropagation>
```

Comment:
Sometimes the compoents already have stop propagation as a property but not all of them so this is most likely the way to do the thing.